src/components/SchedulePage/ProgramTable.tsx

```diff
@@ -0,0 +1,164 @@
+import styled from "styled-components";
+import { ScheduleTable } from "./Data/DayInterface";
+import { MOBILE_BREAKPOINT } from "../../styles/GlobalStyle";
+// Define the interface for props
+interface StyledProps {
+  bgColor: string;
+}
+
+interface ProgramTableProps {
+  day: ScheduleTable;
+}
+
+function ProgramTable({ day }: ProgramTableProps) {
+  return (
+    <Wrapper>
+      <Heading>
+        <Date>{day.date}</Date>
+        <Text>Displayed time zone: </Text>
+        <Timezone>India(Chennai, Kolkata, Mumbai, New Delhi) </Timezone>
+        change
+      </Heading>
+      {day.scheduleTableChildren.map((sessionData, index) => (
+        <Session key={index} bgColor={sessionData.color}>
+          <SessionHeader>
+            <IntialContent>
+              <SessionTime>{sessionData.mainTime}</SessionTime>
+              <SessionType>{sessionData.leading}</SessionType>
+            </IntialContent>
+            <SessionCategory>{sessionData.trailing}</SessionCategory>
+          </SessionHeader>
+
+          {sessionData.subChildren.map((subChild, keyIndex) => (
+            <Section key={keyIndex} bgColor={subChild.color}>
+              <Duration>{subChild.time}</Duration>
```

Comment:
Make duration and Session time same fixed width for different screen sizes so that the text is aligned correctly